test/unit_test/workflows/mock_data/transformer_dataset.json

```diff
@@ -0,0 +1,115 @@
+{
+    "verbose": true,
```

Comment:
I really like this setup for workflow unit testing.
We should add such test for each Olive feature individually, of course separate PR.